WIP : Refactor models of the API

Inao0 opened the pull request:
Fix #7 
And  add some documentation for deployment for development Fix #2

Inao0 committed:
:boom: Refactor models of the API (Fix #7)

Inao0 committed:
:pencil: Add README.md and documentation for dev deployment (Fix #2)

Shysto reviewed (approved):
Ça serait possible de stocker dans l'API un champ technique et dimensions pour une oeuvre? 
Sinon, les changements effectués sont parfaits ! :)

Inao0 committed:
:boom: Add hauteur and largeur fields in API models and change date to annee
Add also Schema to the models in order to add descriptions and constraints for some fields

Inao0 commented:
:ok_hand: les champs technique et dimensions ont été ajoutés ainsi que des descriptions et des contraintes sur certains champs. date a été transformée en un champs annee.

Inao0 committed:
:ok_hand: Change size unit to cm

ubrisson reviewed (approved):
Ok pour moi
Peut-être rajouter les package pythons nécéssaires (qui sont dans required.txt) dans le readme.md ?
